### PR TITLE
5_3_2 to 5_3_2b

### PR DESCRIPTION
**Pull request by nolo16**

### CODE REVIEW

**Review by yk1512 — APPROVED** [review]

File: script/Arcueid/Day 5/05_00_ARC05_3_2B.txt [hunk 1]
```diff
@@ -136,12 +136,13 @@ Wouldn't that kind of temperature be enough to melt even a rock?
 [sha:d11227d46b8879b3c5bc5800a20826f59174f6ce]{
 -- Page 256, Offset 9228.
 -- アレは地表にかかっているだけの軽度の汚染だから、志貴が物陰から出る時、ちょっと気合いを出して吹き飛ばしてあげる。
+// idk what 汚染 means in nasu vampire speak
```
[the remark this answers is not in the document]
Comment (nolo16) [reply script/Arcueid/Day 5/05_00_ARC05_3_2B.txt:139]: would "Their contamination is only strong enough to cover the surface..." work?

File: script/Arcueid/Day 5/05_00_ARC05_3_2B.txt [hunk 2]
```diff
@@ -121,12 +121,12 @@ After taking that hit, I figured that Vlov's flame was somewhere around 3000℃.
...
 -- Page 255, Offset 9225.
 -- いや、3000℃を“程度”って、おかしくないか？
-No, isn't it weird to say a number like 3000℃ so casually?
+No, why are you throwing around a huge number like 3000℃ so casually?
 }
 [sha:bb54d42ec501d1e20e6508db8bbb97ea0737d050]{
 -- Page 255, Offset 9226.
 -- 岩でも熔ける温度だよな、それ？
-Wouldn't that kind of temperature be enough to melt even a rock?
+Isn't that so hot it could even melt a rock?
```
Comment (rschlaikjer) [inline script/Arcueid/Day 5/05_00_ARC05_3_2B.txt:129]: Isn't that hot enough to melt rocks?

File: script/Arcueid/Day 5/05_00_ARC05_3_2B.txt [hunk 3]
```diff
@@ -151,7 +152,7 @@ Although I couldn't do it back in the hotel since you were behind me."
 [sha:d06becc9b53d57489240a5ce4cc27edde899545e]{
 -- Page 257, Offset 9231.
 -- ずいぶんとイージーかつ適当なニュアンスを感じるのだが、事実、あの時はこいつに助けられていた。
-It feels a little too easy, but the fact is that she already saved my life once.
+It feels a bit too easy, but the fact is that she has already saved my life once.
```
Comment (rschlaikjer) [inline script/Arcueid/Day 5/05_00_ARC05_3_2B.txt:155]: she's

File: script/Arcueid/Day 5/05_00_ARC05_3_2B.txt [hunk 4]
```diff
@@ -186,21 +187,22 @@ I won't be able to defeat him with my strength tonight, but I can at least put o
 [sha:848e9ebf26d618c10c4d76c015464b74b56f58ec]{
 -- Page 260, Offset 9238.
 -- 直接組み伏せるとか無駄すぎるし。今まで一度もしたコトないし。する必要性が感じられないし」
-It would be futile to hold him down directly. Besides, I've never done it before, and I don't feel like starting now."
+It would be futile to hold him down directly. Besides, I've never done it before, nor do I feel the need to do it."
```
Comment (rschlaikjer) [inline script/Arcueid/Day 5/05_00_ARC05_3_2B.txt:190]: Kinda like the idiomatic ver, though it is maybe a liberty. If we keep the new ver, I'd drop the `do it` on the end.

File: script/Arcueid/Day 5/05_00_ARC05_3_2B.txt [hunk 5]
```diff
@@ -233,32 +235,32 @@ Wait, where did these chains even come from!?
 [sha:4ab6e51bcc6108923a7fcc61824fbaec3a0ba821]{
 -- Page 263, Offset 9247. Glued. Mods: @x.
 -- 強度は……んー、今なら一本あたり100トンぐらいまでかな？　とにかく何十本と重ねがけするから、動きだけなら封じられる」
- As for how strong they are... at the moment, each chain could probably hold about a hundred tons? I'll pile a couple dozen of these on him, which should cut off his movement at the very least."
+ As for their strength... at the moment, each chain could probably hold about a hundred tons? I'll pile a couple dozen of these on him, which should cut off his movement at the very least."
```
Comment (rschlaikjer) [inline script/Arcueid/Day 5/05_00_ARC05_3_2B.txt:237]: Prefer original

File: script/Arcueid/Day 5/05_00_ARC05_3_2B.txt [hunk 6]
```diff
@@ -233,32 +235,32 @@ Wait, where did these chains even come from!?
...
 }
 [sha:55e1425de0fdb1b962ed6fbd4577f637dfee71e9]{
 -- Page 264, Offset 9248.
 -- 「鎖の拘束は手足をメインにして胴体と頭は控えめにするから、志貴はヴローヴの胴体の“死”を切ればいい」
-"I'll mainly restrain his limbs and leave some space on his head and torso, so that's where you should aim when you go to cut one of his lines of death."
+"For the most part, I'll be restraining his limbs, but I'll leave some space in his torso and head. So when you're about to cut one of his lines of death, you should aim there."
```
[the remark this answers is not in the document]
Comment (rschlaikjer) [reply script/Arcueid/Day 5/05_00_ARC05_3_2B.txt]: Might do a little more massaging here
> "I'll try and restrain him by just wrapping chains around his limbs, and leave his torso and head visible. That way you have something to aim for when you go in to cut his lines of death."

File: script/Arcueid/Day 5/05_00_ARC05_3_2B.txt [hunk 7]
```diff
@@ -478,7 +480,7 @@ This apartment is no different than the hotel. If there is some place with very
 [sha:25f8c9d9732809f3375b6fcc1fbbe0de9a0a01e9]{
 -- Page 284, Offset 9296. Glued. Mods: @x.
 -- ……それでも通りかかってしまう人がいたのなら、それは純粋に、その人に運がなかっただけの話でしょう」
- ―――But if there do happen to be people passing by, it would be purely their own bad luck."
+ ―――But if there do happen to be people passing by, then all that means is they simply happened to have bad luck."
```
Comment (rschlaikjer) [inline script/Arcueid/Day 5/05_00_ARC05_3_2B.txt:482]: Feels slightly overqualified, maybe just
> ...passing by, then that's just their own bad luck.

File: script/Arcueid/Day 5/05_00_ARC05_3_2B.txt (hunk 1 above)
[the remark this answers is not in the document]
Comment (yk1512) [reply script/Arcueid/Day 5/05_00_ARC05_3_2B.txt:139]: works for me

File: script/Arcueid/Day 5/05_00_ARC05_3_2B.txt (hunk 4 above)
Comment (yk1512) [reply script/Arcueid/Day 5/05_00_ARC05_3_2B.txt:190]: _I feel the need for speed_

**Pull request merged**